quail/processes/wps_climdexInput_csv.py

```diff
@@ -234,19 +235,27 @@ def _handler(self, request, response):
             log_level=loglevel,
             process_step="process",
         )
-        ci = climdex.climdexInput_csv(
-            **params,
-            base_range=robjects.r(base_range),
-            na_strings=na_strings,
-            cal=robjects.r(f"'{cal}'"),
-            n=n,
-            northern_hemisphere=northern_hemisphere,
-            quantiles=robjects.r(quantiles),
-            temp_qtiles=robjects.r(temp_qtiles),
-            prec_qtiles=robjects.r(prec_qtiles),
-            max_missing_days=robjects.r(max_missing_days),
-            min_base_data_fraction_present=min_base_data_fraction_present,
-        )
+
+        try:
+            ci = climdex.climdexInput_csv(
+                **params,
+                base_range=robjects.r(base_range),
+                na_strings=na_strings,
+                cal=robjects.r(f"'{cal}'"),
+                n=n,
+                northern_hemisphere=northern_hemisphere,
+                quantiles=robjects.r(quantiles),
+                temp_qtiles=robjects.r(temp_qtiles),
+                prec_qtiles=robjects.r(prec_qtiles),
+                max_missing_days=robjects.r(max_missing_days),
+                min_base_data_fraction_present=min_base_data_fraction_present,
+            )
+        except RRuntimeError as e:
+            err = ProcessError(msg=e)
+            if err.message == "Sorry, process failed. Please check server error log.":
+                raise ProcessError(msg=f"Failure running climdex.climdexInput.csv()")
+            else:
+                raise err
```

Comment:
Why are we checking and changing the error message here?

Reply:
`processError` will automatically change the error message to "Sorry, process failed. Please check server error log." if any "not allowed" characters are in the msg passed to `processError()` so this is kind of a catch all, so If we don't know what the error message will be, print it if possible, but if not at least specify that the process failed during this function run